units/en/unit2/3.md

```diff
@@ -0,0 +1,235 @@
+# Automatic Benchmarks
+
+Automatic benchmarks serve as standardized tools for evaluating language models across different tasks and capabilities. While they provide a useful starting point for understanding model performance, it's important to recognize that they represent only one piece of a comprehensive evaluation strategy.
+
+## Understanding Automatic Benchmarks
+
+Automatic benchmarks typically consist of curated datasets with predefined tasks and evaluation metrics. These benchmarks aim to assess various aspects of model capability, from basic language understanding to complex reasoning. The key advantage of using automatic benchmarks is their standardization - they allow for consistent comparison across different models and provide reproducible results.
+
+However, it's crucial to understand that benchmark performance doesn't always translate directly to real-world effectiveness. A model that excels at academic benchmarks may still struggle with specific domain applications or practical use cases.
+
+## Major Benchmarks and Their Focus
+
+### General Knowledge: MMLU
+
+**Massive Multitask Language Understanding (MMLU)** tests knowledge across 57 subjects, from STEM to humanities. It evaluates:
```

Comment:
Could be nice linking papers for the benchmarks for those that want to learn more